src/common/config/init.js

```diff
@@ -3,15 +3,95 @@
+ * @property {Object} [harvest]
+ * @property {number} [harvest.interval] - The interval in seconds at which the agent will send out data. It's not recommended to change this value.
+ * @property {Object} [jserrors]
+ * @property {boolean} [jserrors.enabled] - Turn on/off the jserrors feature (on by default).
+ * @property {boolean} [jserrors.autoStart] - If true, the agent will automatically start the jserrors feature. Otherwise, it will be in a deferred state until the `start` API method is called.
+ * @property {Object} [logging]
+ * @property {boolean} [logging.enabled] - Turn on/off the logging feature (on by default).
+ * @property {boolean} [logging.autoStart] - If true, the agent will automatically start the logging feature. Otherwise, it will be in a deferred state until the `start` API method is called.
+ * @property {Object} [metrics]
+ * @property {boolean} [metrics.enabled] - Turn on/off the metrics feature (on by default).
+ * @property {boolean} [metrics.autoStart] - If true, the agent will automatically start the metrics feature. Otherwise, it will be in a deferred state until the `start` API method is called.
+ * @property {Array<Object>} [obfuscate] - Array of regexp and corresponding replacement patterns for obfuscating data.
+ * @property {Object} [page_action]
+ * @property {boolean} [page_action.enabled] - Must be true to allow PageAction events to be captured.
+ * @property {Object} [page_view_event]
+ * @property {boolean} [page_view_event.enabled] - This setting is ignored! PageViewEvent is always enabled by force.
+ * @property {boolean} [page_view_event.autoStart] - If true, the agent will automatically send the RUM request. Otherwise, it will be in a deferred state until the `start` API method is called.
+ * @property {Object} [page_view_timing]
+ * @property {boolean} [page_view_timing.enabled] - Turn on/off the page view timing feature (on by default).
+ * @property {boolean} [page_view_timing.autoStart] - If true, the agent will automatically start the page view timing feature. Otherwise, it will be in a deferred state until the `start` API method is called.
+ * @property {Object} [performance]
+ * @property {boolean} [performance.capture_marks] - If true, the agent will capture PerformanceMark events.
+ * @property {boolean} [performance.capture_measures] - If true, the agent will capture PerformanceMeasure events.
+ * @property {boolean} [performance.capture_detail] - If true, `BrowserPerformance` events from marks and measures will include, as attribute(s), the `detail` metadata provided to `markOptions` and `measureOptions`.
+ * @property {Object} [performance.resources]
+ * @property {boolean} [performance.resources.enabled] - If true, the agent will capture PerformanceResourceTiming entries.
+ * @property {Array<string>} [performance.resources.asset_types] - Array of `initiatorType` strings to filter the desired ResourceTiming entries. By default, all resource types are captured.
```

Comment:
This should note that an empty array also captures all types

Reply:
```suggestion
 * @property {Array<string>} [performance.resources.asset_types] - Array of `initiatorType` strings to filter the desired ResourceTiming entries. By default, all resource types are captured, which is also the case if an empty array is provided.
```